Cython/Compiler/Nodes.py

```diff
@@ -6518,7 +6518,7 @@ class InPlaceAssignmentNode(AssignmentNode):
     #
     #  This code is a bit tricky because in order to obey Python
     #  semantics the sub-expressions (e.g. indices) of the lhs must
-    #  not be evaluated twice. So we must re-use the values calculated
+    #  not be evaluated twice. So we must reuse the values calculated
```

Comment:
As far as I can tell both forms are right so not sure I want to change this (I kind of prefer the hypenated form, although wouldn't correct the other form if it was already there)